dotnet/generate-workloadmanifest-json.csharp

```diff
@@ -50,11 +50,13 @@ using (TextWriter writer = new StreamWriter (outputPath)) {
 	writer.WriteLine ($"			\"description\": \".NET SDK Workload for building {platform} applications.\",");
 	writer.WriteLine ($"			\"packs\": [");
 	foreach (var tfm in allApiVersions) {
-		writer.WriteLine ($"				\"Microsoft.{platform}.Sdk.{tfm}\",");
+		var sdkTfm = tfm == "net8.0" ? "net8" : tfm;
```

Comment:
Do we also need this kind of change in the workload targets file? We encountered this issue early when inserting preview 4 and it looks like this was the fix we went with https://github.com/xamarin/xamarin-macios/pull/20516/files, but maybe it didn't make it back to the net9.0 branch?

Reply:
Ah, I'd completely forgotten about that change.

I reverted the changes from today and just pushed those changes instead.